service/history/historyEngineInterfaces.go

```diff
@@ -57,43 +58,29 @@ type (
 
 	queueAckMgr interface {
 		getFinishedChan() <-chan struct{}
-		readQueueTasks() ([]tasks.Task, bool, error)
-		completeQueueTask(taskID int64)
+		readQueueTasks() ([]queues.Executable, bool, error)
```

Comment:
just want to confirm, this `readQueueTasks() ([]queues.Executable, bool, error)` is temporary?

Reply:
Yes. All interfaces defined in this file will be removed after multicursor queue is done.